### PR TITLE
Fix style of permission view buttons

### PR DESCRIPTION
**Pull request by pusztaienike**

### CODE REVIEW

File: apps/sensenet/src/components/view-controls/permission-view.tsx [hunk 1]
```diff
@@ -42,9 +42,11 @@ const useStyles = makeStyles((theme: Theme) => {
     },
     title: {
       fontSize: '20px',
+      paddingRight: '10px',
     },
-    textBolder: {
+    contentName: {
       fontWeight: 500,
+      wordBreak: 'break-word',
```
Comment (taki9) [inline apps/sensenet/src/components/view-controls/permission-view.tsx:49]: it is working with the default value (normal) of wordBreak, so you can delete this

**Review by taki9 — CHANGES_REQUESTED** [review]

File: apps/sensenet/src/components/view-controls/permission-view.tsx (hunk 1 above)
Comment (pusztaienike) [reply apps/sensenet/src/components/view-controls/permission-view.tsx:49]: but it is not working with some display size 
I think it is better with break
permission_button

**Review by herflis — APPROVED** [review]

File: apps/sensenet/src/components/view-controls/permission-view.tsx [hunk 2]
```diff
@@ -42,8 +44,14 @@ const useStyles = makeStyles((theme: Theme) => {
     },
     title: {
       fontSize: '20px',
+      paddingRight: '10px',
+      width: `calc(100% - ${permissionActionbuttonsWidth}px)`,
```
Comment (taki9) [inline apps/sensenet/src/components/view-controls/permission-view.tsx:48]: I think you could remove width and margin set, and get the same result without magic numbers.

**Review by taki9 — CHANGES_REQUESTED** [review]

File: apps/sensenet/src/components/view-controls/permission-view.tsx (hunk 2 above)
Comment (pusztaienike) [reply apps/sensenet/src/components/view-controls/permission-view.tsx:48]: without these it seems like the screenshot:
image

**Pull request merged**